--- eman2-2.21a/examples/fftspeed.py	2018-02-20 15:37:14.000000000 -0800
+++ patch-files/fftspeed.py	2018-03-12 18:59:49.041623400 -0700
@@ -58,7 +58,7 @@
 
 for dim in xrange(2,4):
 	times=[]
-	for size in xrange(20,514,2):
+	for size in xrange(20,128,2):
 		if dim==2 : 
 			img=EMData(size,size,1)
 			reps=int(200000/size)
@@ -92,4 +92,5 @@
 	print("\n")
 
 	out.write(str(times2)+"\n")
+        print("ALL TESTS RAN")
 
