--- relion-1.4/src/Healpix_2.15a/cxxutils.cc	2014-12-02 05:41:52.000000000 -0800
+++ patch-files/cxxutils.cc	2018-04-05 16:12:09.269126065 -0700
@@ -55,7 +55,7 @@
 bool file_present (const string &filename)
   {
   ifstream dummy(filename.c_str());
-  return dummy;
+  return (bool) dummy;
   }
 
 void assert_present (const string &filename)
@@ -261,7 +261,7 @@
   int lineno=0;
   dict.clear();
   ifstream inp(filename.c_str());
-  planck_assert (inp,"Could not open parameter file "+filename);
+  planck_assert ((bool) inp,"Could not open parameter file "+filename);
   while (inp)
     {
     string line;
